### ISSUE TITLE
Follow issue about the database

### ISSUE THREAD

**Issue by ZhangAngus** [MIT-LCP/mimic-code#647]
Thank you for your team maintaining the MIMIC database. There are a few questions happened to me.

1. Once Alistair said that MIMIC always follow 90 days at least after discharging from hospital, but in issue 191 ([https://github.com/MIT-LCP/mimic-code/issues/191](https://github.com/MIT-LCP/mimic-code/issues/191)), Tom said it was 150 days. Which one is correct?
2. What's the defined day of the start date line of following, the day admitted or discharge?
Thank you for taking the time to answer my questions, wish you a lovely day.

**Comment by alistairewj** [comment]
1. The worst-case censoring for date of death is 90 days from ICU admission.
2. I'm not sure I understand - could you rephrase the question?

**Comment by tompollard** [comment]
> The worst-case censoring for date of death is 90 days from ICU admission.

I thought we'd checked this and it was 150 days (which is why I'd said that!).

**Comment by alistairewj** [comment]
Agree to disagree? :) Let's double check..

**Comment by tompollard** [comment]
Okay, I have looked at this in MIMIC-III (v1.4). The census dates are detailed below:

```sql
SELECT :max_ssn_date - max(a.dischtime) AS datediff
FROM patients p
LEFT JOIN admissions a
ON p.subject_id = a.subject_id
WHERE p.dod is null;
-- 82 days from hospital discharge


SELECT :max_ssn_date - max(a.admittime) AS datediff
FROM patients p
LEFT JOIN admissions a
ON p.subject_id = a.subject_id
WHERE p.dod is null;
-- 118 days from hospital admission


SELECT :max_ssn_date - max(i.intime) AS datediff
FROM patients p
LEFT JOIN icustays i
ON p.subject_id = i.subject_id
WHERE p.dod is null;
-- 107 days from ICU intime


SELECT :max_ssn_date - max(i.outtime) AS datediff
FROM patients p
LEFT JOIN icustays i
ON p.subject_id = i.subject_id
WHERE p.dod is null;
-- 102 days from ICU outtime
```

It is possible that the case was different for v1.3, but I haven't checked. The question would be whether changes between releases affected the number. If not, then the 150 days is a mistake on my behalf and I apologize (perhaps a result of looking at the date of a date of death dataset that hadn't been merged).

**Comment by ZhangAngus** [comment]
> 1. The worst-case censoring for date of death is 90 days from ICU admission.
> 2. I'm not sure I understand - could you rephrase the question?

Sorry for my confusing second question.
The second is about the start day of 90 days. We can see codes from Tom
> -- 82 days from hospital discharge
> -- 118 days from hospital admission
> -- 107 days from ICU intime
> -- 102 days from ICU outtime

There will be days from latest hospital discharge, hospital admission, ICU intime and ICU outtime.
Which one is the start day of 90 days.

And according to your reply to the first question, it seems referring to the latest ICU admission time, which is the latest ICU intime.

Am I right?

**Comment by alistairewj** [comment]
Yes. Tom calculated it exactly.
